src/main/webapp/home.jsp

```diff
@@ -30,6 +30,10 @@
               <li class="nav-item">
                   <a class="nav-link" href="stats.html">Stats</a>
                 </li>
+              <li class="nav-item">
+                  <a class="nav-link" href="location-form.html">Location Form</a>
+                </li>
+
                 <%
```

Comment:
I believe there should also be a %>

Reply:
There is one on line 40 (formerly line 36), you just have to make the hidden lines visible :-)